content/en/Getting started/What to Expect/_index.md

```diff
@@ -62,7 +62,7 @@ results. Here's what you can expect:
    However, we report _all_ findings. For more information, see the following blog post
    on how you can [Customize Your Pentest
    Reports](https://cobalt.io/blog/cobalt-platform-deep-dive-customize-your-pentest-reports-per-your-needs).
-1. Within days, our pentesters share a more formal pentest report.
+1. <a id="report-timing">Within days, our pentesters share a more formal pentest report.
```

Comment:
Added to support a link from Slack. Otherwise, the link starts with "Step 1", and readers might not see when we'd share the "more formal pentest report"